sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceClientBuilder.java

```diff
@@ -351,18 +351,18 @@ public TableServiceClientBuilder serviceVersion(TablesServiceVersion version) {
     /**
      * Sets the request retry options for all the requests made through the client.
      *
-     * @param retryOptions {@link RequestRetryOptions}.
+     * @param retryPolicy {@link RetryPolicy}.
      *
      * @return The updated {@link TableServiceClientBuilder}.
      *
-     * @throws NullPointerException if {@code retryOptions} is {@code null}.
+     * @throws NullPointerException if {@code retryPolicy} is {@code null}.
      */
-    public TableServiceClientBuilder retryOptions(RequestRetryOptions retryOptions) {
-        if (retryOptions == null) {
-            throw logger.logExceptionAsError(new NullPointerException("'retryOptions' cannot be null."));
+    public TableServiceClientBuilder retryPolicy(RetryPolicy retryPolicy) {
+        if (retryPolicy == null) {
+            throw logger.logExceptionAsError(new NullPointerException("'retryPolicy' cannot be null."));
```

Comment:
`retryPolicy` can be set to null in which case we'll fall back to default retry policy.

Reply:
We'll do this in #21368.